Astral Drift/Assets/Nicolaas/Scripts/HumanPickupable.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HumanPickupable : Pickupable
+{
+    // A model is needed to not get issues with the collider.
+    [SerializeField] private GameObject model;
+
+    [SerializeField] private int scoreIncrease = 10;
+    [SerializeField] private float pickupTime;
+    [SerializeField] [Range(0, 1)] private float scaleWhenCollected; // This is a percentage value, thus ranging from 0 to 1
+
+    private Vector3 originalScale;
+    private float timer;
+    private bool beingPickedUp;
+
+    private void Start()
+    {
+        // This causes it to do less math for fixed update. Not done the other way around to make serialize field easier to understand.

```

Comment:
Causes _what_ to do less math? I also can't make sense of your second sentence in the comment.